### ISSUE TITLE
SpellChecker for CodeMirror

### ISSUE THREAD

**Issue by Belphemur** [TiddlyWiki/TiddlyWiki5#1630]
Hi,

I have the same problem as #1298. I know I can disable CodeMirror, but I found this: 
http://stackoverflow.com/questions/12343922/codemirror-with-spell-checker

Do you think you could implement it in the Node.js version that use CodeMirror ?

**Comment by Jermolene** [comment]
Hi @Belphemur 

> Do you think you could implement it in the Node.js version that use CodeMirror ?

It looks like it could be integrated with the TiddlyWiki CodeMirror plugin. It's not something that I could commit to doing, I'm afraid, but I'd be happy to support another developer working on it.

> Do you think you could implement it in the Node.js version that use CodeMirror ?

By the way, the CodeMirror plugin can be used in the standalone HTML file configuration of TiddlyWiki. It's included in the client-server configuration by default.

**Comment by Belphemur** [comment]
Do you think you could instead make the default type or text/vnd.tiddlywiki not using codemirror by default ?

**Comment by Jermolene** [comment]
> Do you think you could instead make the default type or text/vnd.tiddlywiki not using codemirror by default ?

Well, we could simply remove CodeMirror from the default plugins included in the `server` edition. Is the motivation so that spell checking works?

**Comment by Belphemur** [comment]
Yes that's the basic motivation. I just thought keeping CodeMirror is good since it provides useful feature if you plan on doing a technical wiki.

**Comment by sukima** [comment]
Sadly I looked into this very problem once. Because of the effort to make more plugins and fix CodeMirror I gave up and now copy paste between my preferred editor and TiddlyWiki. I found my main editor (Vim) had better spell checking then the browser and obviously better Vim key bindings then CodeMirror.

I'd love to see a typo.js plugin for the CodeMirrir plugin made because least I could get my Vi and spell checking done on computers that don't have Vim installed.

**Comment by pmario** [comment]
Close in favour of 

- #6927
